apps/chat/src/components/Chatbar/ChatFolders.tsx

```diff
@@ -110,6 +110,14 @@ const ChatFolderTemplate = ({
     ConversationsSelectors.selectSelectedItems,
   );
 
+  const emptyFoldersIds = useAppSelector(
+    ConversationsSelectors.selectEmptyFolderIds,
+  );
+
+  const isFolderEmpty = useAppSelector((state) =>
+    ConversationsSelectors.selectIsFolderEmpty(state, folder.id),
+  );
```

Comment:
Why not `const isFolderEmpty = emptyFoldersIds.includes(folder.id);`?

Reply:
probably with `useMemo`

Reply:
I think it is better to have separate selector for this for the reusability.